README.md

```diff
@@ -2,8 +2,72 @@
 
 [![Netlify Status](https://api.netlify.com/api/v1/badges/a3b0b564-6d90-4bbf-9b5f-2f5fd46d5a97/deploy-status)](https://app.netlify.com/sites/smart-city-dashboard/deploys)
 
+This project contains the source code of the frontend of the smart city dashboard. The display can connect to the [backend](https://github.com/reedu-reengineering-education/smart-city-dashboard-backend) and show different kinds of environmental and smart city related data. The dashboard contains three different pages: dashboard view, map view and an info page.
```

Comment:
shows

Reply:
Resolved in [ca1fb3b](https://github.com/reedu-reengineering-education/smart-city-dashboard/pull/15/commits/ca1fb3b0457e9f148eef04857de7522da4ca8815)